upgrade/pack_builder/SparseVector.hpp

```diff
@@ -60,6 +66,13 @@ namespace application
         : position(position)
     {}
 
+    inline char const* OverwriteException::what() const
+    {
+        std::stringstream ss;
+        ss << "Contents specified twice for memory location at address 0x" << std::hex << std::setw(8) << std::setfill('0') << position;
+        return ss.str().c_str();
```

Comment:
You are returning a pointer to local storage here. Referring it later will lead to undefined behavior.